docs/testrunner-toolkit/ide-integrations/vscode.md

```diff
@@ -0,0 +1,41 @@
+---
+id: vscode
+title: saucectl with visual studio code
+sidebar_label: Visual Studio Code
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+## What You'll Need
+
+* [Visual Studio Code](https://code.visualstudio.com/)
+* [YAML Extension](https://marketplace.visualstudio.com/items?itemName=redhat.vscode-yaml)
+
+## JSON Schema Validation
+
+Visual Studio Code can help you edit the `saucectl` configuration files by providing helpful suggestions, autocompletion and docs; all without having to leave the IDE.
+Our JSON schema is published to the [JSON Schema Store](https://www.schemastore.org/json/), which is made available to Visual Studio Code.
+
+By default, JSON Schema validation *only* works with JSON files, not YAML. So make sure to have the YAML extension installed or this will not work.
```

Comment:
```suggestion
:::caution
By default, JSON Schema validation *only* works with JSON files, not YAML. So make sure to have the YAML extension installed or this will not work.
:::
```

Reply:
For your convenience. Or if you're doing it locally, just ignore me. :)